solidity/test/contracts/RandomBeaconStub.sol

```diff
@@ -1,10 +1,10 @@
 pragma solidity ^0.5.4;
 
-import "@keep-network/keep-core/contracts/IRandomBeacon.sol";
+//import "@keep-network/keep-core/contracts/IRandomBeacon.sol";
 
 /// @title Random Beacon Service Stub
 /// @dev This contract is for testing purposes only.
-contract RandomBeaconStub is IRandomBeacon {
+contract RandomBeaconStub /*is IRandomBeacon*/ {
```

Comment:
I hate what I did here but this can't compile until we release a new `keep-core` version or fix `-pre` deployments. I am seriously considering turning `is IRandomBeacon` into a TODO. Thoughts?

Reply:
We can't merge this PR without `keep-core` changes being released. The release will solve the problem you mentioned above.

Reply:
No, this is not a good idea. The build result will depend on the currently cached `keep-core` version. Let's hold on with merging this PR until we release a new `keep-core`.

Reply:
@nkuba We commented here at the same time :)

Reply:
![](https://media.giphy.com/media/11sXw14uVtghgI/giphy.gif)

Reply:
It's back here: https://github.com/keep-network/keep-ecdsa/pull/339/commits/85fa8f073b8adb9761f526fc9997c65b5489b84b